src/plugins/auto/plugin.cpp

```diff
@@ -311,6 +311,12 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
         AutoContext context;
         std::map<std::string, std::string> filterConfig;
         CheckConfig(fullConfig, context, filterConfig);
+
+        //for the case of -d "AUTO"
+        if (priorities == fullConfig.end()) {
```

Comment:
Consider -d AUTO:-VPU  case
https://github.com/openvinotoolkit/openvino/pull/11516 enableDevicePriority

Reply:
or we can check the different device priority here directly to see if priority have been applied.